tests/tools/oesign/test-enclave/host/host.c

```diff
@@ -19,12 +39,32 @@ int main(int argc, const char* argv[])
         oe_put_err("Usage: %s enclave_image_path\n", argv[0]);
     }
 
+    /* Load the ELF image */
+    if ((result = oe_load_enclave_image(argv[1], &oeimage)) != OE_OK)
+    {
+        oe_put_err("oe_load_enclave_image(): result=%u", result);
+    }
+
+    /* Load the SGX enclave properties */
+    if ((result = oe_sgx_load_enclave_properties(
+             &oeimage, OE_INFO_SECTION_NAME, &properties)) != OE_OK)
+    {
+        oe_put_err("oe_sgx_load_enclave_properties(): result=%u", result);
+    }
+
     // Create the enclave
     const uint32_t flags = oe_get_create_flags();
 
     if ((result = oe_create_oesign_test_enclave(
              argv[1], OE_ENCLAVE_TYPE_AUTO, flags, NULL, 0, &enclave)) != OE_OK)
     {
+        if (!_is_kss_supported() &&
+            properties.config.attributes & SGX_FLAGS_KSS)
+        {
+            printf("Skipping Enclave creation with Kss feature \
```

Comment:
This actually prints a lot of spaces between "feature" and "if". Do this instead:

    printf("Skipping Enclave creation with Kss feature "
            "if hardware doesn't support.\n");